Formula/graph-tool.rb

```diff
@@ -16,7 +16,7 @@ class GraphTool < Formula
   depends_on "boost"
   depends_on "boost-python3"
   depends_on "cairomm"
-  depends_on "cgal"
+  depends_on "maxgimeno/core/cgal"
```

Comment:
This was, I think, pushed by mistake

Reply:
Indeed. It will be fixed the second we manage to make the formula work.